packages/light-apps/src/AddAccount/AddAccount.tsx

```diff
@@ -2,41 +2,48 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { AppContext } from '@substrate/ui-common';
-import { Menu, WalletCard } from '@substrate/ui-components';
 import React from 'react';
 import { RouteComponentProps, Link, Switch, Route, Redirect } from 'react-router-dom';
+import { Container } from 'semantic-ui-react';
+import { AppContext } from '@substrate/ui-common';
```

Comment:
I know apps does like this, it bothered me b/c `['@s', 's'].sort()` shows `@s` first, but we can change it to this, as long as it's consistent.

Reply:
I feel the same (would prefer `@s` to come first, but it's ok if we change this so long as it's consistent). Note that the current styleguide https://github.com/paritytech/substrate-light-ui/blob/master/CONTRIBUTING.md#styleguides says that `@s` comes before s